taccsite_cms/static/site_cms/css/src/site.css

```diff
@@ -6,8 +6,8 @@
 /* SETTINGS */
 /* Shared with Portal */
 @import url("_imports/settings/border.css");
-@import url("_imports/settings/color.css");
 /* Unique to CMS */
+@import url("_imports/settings/color.css");
```

Comment:
__Undo this change.__ This change is for global typography (upcoming PR), not breadcrumbs.